src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -122,6 +122,17 @@ public function load(array $configs, ContainerBuilder $container)
             $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
         }
 
+        if ($container->fileExists($dir = $container->getParameter('kernel.project_dir').'/templates', false)) {
+            $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
```

Comment:
I got the point, just that I liked the idea to remove the old convention in favor of explicit configuration. I thought in this in the other way around (add the feature to the configured `templates` path) but there is not way to know what is it right now. So I guess rely on the approved structure should be the path. :+1: